src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -219,11 +214,13 @@ export const RuntimeTransactionDetailView: FC<{
             <>
               <dt>{t('common.to')}</dt>
               <dd>
-                <AccountLink
-                  scope={transaction}
-                  address={(transaction?.to_eth || transaction?.to) as string}
-                />
-                <CopyToClipboard value={(transaction?.to_eth || transaction?.to) as string} />
+                <Box sx={{ display: 'inline-flex', alignItems: 'center' }}>
```

Comment:
Related https://github.com/oasisprotocol/explorer/issues/1535

"To" field does resize, but copy button was in its own line
![image](https://github.com/user-attachments/assets/1f6f67c3-2b1b-450a-96ec-38b1baa743dc)

Reply:
Moving out to a separate PR